lms/djangoapps/onboarding/admin.py

```diff
@@ -76,6 +76,7 @@ class CurrencyAdmin(admin.ModelAdmin):
 
 class EmailPreferenceAdmin(admin.ModelAdmin):
     list_display = ('user', 'opt_in', 'modified_time', )
+    raw_id_fields = ('user',)
```

Comment:
How about to override `admin.ModelAdmin` and use that everywhere?

Reply:
Very good suggestion, But I think it isn't that much big thing, It's already  single line simple thing that also handles different name then user like responsible_user and admin.

Reply:
@zeshanarifios I think you can also you mixin classes.